supertokens_python/recipe/session/session_class.py

```diff
@@ -33,6 +35,9 @@ async def get_session_data(
         session_info = await self.recipe_implementation.get_session_information(
             self.session_handle, user_context
         )
+        if session_info is None:
+            raise_unauthorised_exception("Session information does not exist.")
```

Comment:
Error message should be `"Session does not exist anymore"` everywhere.

Reply:
done